lib/flow/get-next-steps-for-user.js

```diff
@@ -25,37 +25,41 @@ const userIsAsruAdmin = profile => {
   return profile.asruUser && profile.asruAdmin;
 };
 
-const taskIsOpen = c => {
-  return flow.open().includes(c.status);
+const taskIsOpen = task => {
+  return flow.open().includes(task.status);
 };
 
-module.exports = (c, profile) => {
-  if (flow.autoForwards(c.status)) {
-    return Promise.resolve(flow.getNextSteps(c));
+module.exports = (task, profile) => {
+  if (flow.autoForwards(task.status)) {
+    return Promise.resolve(flow.getNextSteps(task));
   }
 
   let nextSteps = [];
 
+  if (task.data.initiatedByAsru && !profile.asruUser) {
+    return Promise.resolve([]);
+  }
```

Comment:
Adding this block ☝️ is the only bit of this file change that _isn't_ `c => task`